datamade/la-metro-councilmatic
# Log sources

## Overview

Print out the sources for each person, event, and bill on their respective detail page.

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/114717958/214418238-6d6226e5-8490-47c4-96f6-18ea250d41b5.png)

## Testing Instructions

 * Navigate to the detail page for a person, event, or bill
 * Open the console to check if it's source has been printed
 * Follow link(s) present

Handles #910
